pom.xml

```diff
@@ -672,6 +710,48 @@
                 <version>${kryo.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.hive</groupId>
+                <artifactId>hive-exec</artifactId>
+                <version>${hive.version}</version>
+                <classifier>core</classifier>
+                <scope>test</scope>
```

Comment:
nit: move to test deps block

Reply:
updated

Reply:
I removed the scope in dependencyManagement, we may need it, will followup it in another pr

https://github.com/apache/amoro/issues/3933

Reply:
It should be fine to remove the test scope in the dependencyManagement. 

<img width="687" height="245" alt="image" src="https://github.com/user-attachments/assets/e281f68a-6110-45c1-96c6-2356640a0633" />